### ISSUE TITLE
No use provider could resolve identifier 

### ISSUE THREAD

**Issue by meghapatil0312** [adobe/aem-component-generator#59]
i am taking data from json using js,servlet and htl in AEM but i am getting below error     
 
org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: org.apache.sling.api.SlingException: Cannot get DefaultSlingScript: No use provider could resolve identifier content/dam/employee/employees.json 
  
 i think there is issue with bundles ( but i have active bundles)in that i cant see my core files and also in target folder there is no data comping from backend only frontend data it there. Can you help me with this?

**Comment by HitmanInWis** [comment]
Hi @meghapatil0312.  Is your project using the default AEM project archetype?  Do yo have other sling models deployed, or is this your first attempt at deploying sling models?

I don't believe your issue would be related to the generator itself, but happy to help if I or one of my colleagues can.

**Comment by meghapatil0312** [comment]
Yes we are using the default AEM project archetype. And in this project i deployed the first sling model.

**Comment by HitmanInWis** [comment]
If you're not seeing the `core` files in your target folder, it sounds like it might have to do with the build command you're using or maybe even an issue in `pom.xml` files.  At any rate, if you're unable to figure out the problem, perhaps a quick screen share could help get this sorted.  Feel free to reach out to me on LinkedIn - https://www.linkedin.com/in/brettbirschbach/

**Comment by HitmanInWis** [comment]
Closing for inactivity.
